library/kafka_consumer_group.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+"""
+Ansible module for consumer group
+"""
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.pycompat24 import get_exception
+from kafka.errors import KafkaError
+
+from ansible.module_utils.kafka_lib_commons import (
+    module_commons, DOCUMENTATION_COMMON, get_manager_from_params,
+    maybe_clean_kafka_ssl_files
+)
+
+ANSIBLE_METADATA = {'metadata_version': '1.0'}
+
+DOCUMENTATION = '''
+---
+module: kafka_consumer_group
+short_description: Interact with kafka consumer groups
+description:
+     - Interact with kafka consumer groups.
+     - Not compatible with Kafka version < 0.11.0.
+author:
+    - Yassine MILHI
+options:
+  topics_partitions:
+    descritption:
+      - 'consumed topics partitions on which action will be performed'
+    required: True
+  consummer_group:
+    description:
+      - 'one consumer group name.'
+    required: True
+''' + DOCUMENTATION_COMMON
+
+EXAMPLES = '''
+    - name:  Delete offset for consumer group
+    kafka_consumer_group:
```

Comment:
Should we add an `action` choice param that would only take `delete` for now (and that we will expand to `delete`, `reset` later so that this module allows to delete consumer group and reset offsets)?

Reply:
Done